orangecontrib/text/widgets/owstatistics.py

```diff
@@ -622,7 +622,7 @@ def _sync_edit_combo(self) -> None:
         default_value = STATISTICS_DEFAULT_VALUE[selected_i]
         self.active_rules[edit_index] = (
             selected_i,
-            default_value or self.active_rules[edit_index][1],
+            default_value,
         )
```

Comment:
Totally agree with this change. I don't have any idea what I wanted to reach here. It doesn't make sense to keep the current value if the function is changed. 

Sorry for nitpicking, but I would now make this one oneliner since it is now short enough:
```
self.active_rules[edit_index] = (selected_i, default_value)
```